utils/address.ts

```diff
@@ -1,6 +1,3 @@
-import { assertValidRailgunAddress } from '@railgun-community/quickstart';
-
 export const shortenAddress = (address: string) => {
-  assertValidRailgunAddress(address); // Will throw error on incorrect value
```

Comment:
Why did we get rid of this?

Reply:
The main reason was if it errors it crashes the app, and we use it in the jsx. So, If we leave it this way, we will need to add some logic within the component, and the check isn't necessary because we are using form validation.

Reply:
It is simpler to remove rather than work around it.